projects/demo/src/modules/components/button/examples/3/index.ts

```diff
@@ -14,8 +14,4 @@ export class TuiButtonExample3 {
     readonly arrow = TUI_ARROW;
 
     open = false;
```

Comment:
probably, you can also drop this line and `[open]="open"` inside template

Reply:
fixed